assembl/static2/js/app/components/debate/brightMirror/instructionView.jsx

```diff
@@ -0,0 +1,61 @@
+// @flow
+import React from 'react';
+
+import Permissions, { connectedUserCan } from '../../../utils/permissions';
+import TopPostFormContainer from '../common/topPostFormContainer';
+import { getIfPhaseCompletedByIdentifier } from '../../../utils/timeline';
+import FictionsList from './fictionsList';
+import InstructionsText from './instructionsText';
+
+type Post = {
+  id: number,
+  subject: string,
+  creationDate: Date,
+  creator: {
+    displayName: string,
+    isDeleted: boolean
+  }
+};
+
+type Props = {
```

Comment:
@cedrtang define an explicit type name

Reply:
When there is only one component per file, and we should normally have one per file, I prefer we name the flow type simply Props and State. This is what we do currently in the code base.